lib/cocoapods/installer/xcode/pods_project_generator/file_references_installer.rb

```diff
@@ -302,6 +303,21 @@ def common_path(paths)
             return result unless result.to_s == '' || result.to_s == '/'
           end
 
+          # Adds the known localization regions to the root of the project
+          #
+          # @param [Array<PBXFileReferences>] file_references the resource file references
+          #
+          def add_known_regions(file_references)
+            pattern = /.*(\/|^)(?<region>.*)\.lproj(\/|$)/
```

Comment:
```suggestion
            pattern = /(\/|^)(?<region>.*?)\.lproj(\/|$)/
```

also extract to a private constant so we can share a single regexp object?

Reply:
I _think_ this doens't work because there can be things before the leading `/`. Example: `Resources/en.lproj`, we don't want `Resources/en`, just `en`. I added the leading `.*` when the unit tests caught this

Reply:
Ah, in that case:

```suggestion
            pattern = /(\/|^)(?<region>[^\/]*?)\.lproj(\/|$)/
```

Reply:
ace